src/components/VSelect/VSelect.js

```diff
@@ -425,6 +429,7 @@ export default {
 
       if (this.$refs.input && this.isAutocomplete) {
         this.$refs.input.focus()
+        setTimeout(() => this.$refs.input.select(), 20)
```

Comment:
Where does 20ms come from?

Reply:
Also you probably don't need the arrow function

Reply:
from trials and errors